src/sdk/Validator.ts

```diff
@@ -105,12 +105,12 @@ export const validate = (application: Application): string[] => {
       );
     } else {
       application.store.medias.forEach((media) => {
-        if (!media.uri) {
+        if (!media.url) {
           issues.push('Uri value is missing');
```

Comment:
This error message needs editing as well.

Reply:
Agree, fixed